themes/socialbase/templates/activity/activity--featured--sky.html.twig

```diff
@@ -0,0 +1 @@
+{% extends "activity.html.twig" %}
```

Comment:
@volodymyr-sydor-lemberg Is it really needed to have this template? It looks for me that we can remove it.

Reply:
@ribel looks like currently, we don't need it, i will remove it and update related PR

Reply:
done